crates/core/src/db/db_metrics/mod.rs

```diff
@@ -57,27 +57,27 @@ metrics_group!(
 
         #[name = spacetime_num_rows_inserted_cumulative]
         #[help = "The cumulative number of rows inserted into a table"]
-        #[labels(txn_type: WorkloadType, db: Address, reducer_or_query: str, table_id: u32)]
+        #[labels(txn_type: WorkloadType, db: Address, reducer_or_query: str, table_id: u32, table_name: str)]
```

Comment:
Note this is purely additive change so that we won't break metrics upon releasing. Once released, the `table_id` label can be safely removed.